pkg/diff/merge.go

```diff
@@ -55,19 +54,25 @@ func (r RowDatas) Less(i, j int) bool {
 				}
 				continue
 			}
+			if strData1 > strData2 {
+				return false
+			}
+			return true
 		} else {
```

Comment:
if block ends with a return statement, we can drop this else and outdent its block